packages/manager/apps/hub/src/index.html

```diff
@@ -32,11 +32,11 @@
                 universe: 'hub',
                 version: 'beta'
             }"
+            data-sidebar-expand="true"
         >
         </ovh-manager-navbar>
-        <div class="d-flex">
+        <div class="d-flex hub-main-view">
```

Comment:
I wonder if we still need this, maybe we can remove and apply the class on the children